runtime/lua/vim/health.lua

```diff
@@ -118,7 +118,10 @@ local function filepath_to_healthcheck(path)
     func = 'health#' .. name .. '#check'
     filetype = 'v'
   else
-    local subpath = path:gsub('.*/lua/', '')
+    local rtp = vim.iter(vim.api.nvim_list_runtime_paths()):find(function(rtp0)
+      return vim.fs.relpath(rtp0, path)
+    end)
+    local subpath = path:gsub('^' .. vim.pesc(rtp .. '/lua/'), '')
```

Comment:
just guessing, but maybe this will fix the windows ci failure: 

```suggestion
    local subpath = path:gsub('^' .. vim.pesc(rtp) .. '[\\/]lua[\\/]', '')
```

example diff of windows ci failure:

<img width="1297" alt="image" src="https://github.com/user-attachments/assets/f3021160-613c-4f81-bba9-1ceb26c06d9b" />

Reply:
The results of `nvim_get_runtime_file`/`nvim_list_runtime_paths` both need to normalized (`\` -> `/`) for windows.